crates/libs/bindgen/src/extensions/mod.rs

```diff
@@ -16,6 +16,7 @@ pub fn gen(def: &TypeDef) -> TokenStream {
         TypeName::Matrix3x2 => matrix3x2::gen(),
         TypeName::Matrix4x4 => matrix4x4::gen(),
         TypeName::WIN32_ERROR => win32_error::gen(),
+        TypeName::IN_ADDR => in_addr::gen(),
```

Comment:
You forgot to declare the `in_addr` module.

Reply:
Yeah, I pushed that commit before you taught me how to rebuild and test.  I've added ipv6 already and I'm looking at how to set the `sin_family` in the sockaddr structs now.